File: C++/Estrutura Repetitiva/Para/Soma_impares.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;

int main()
{

    int i, x, y, troca, soma;

    cout << "Digite dois numeros: " << endl;
    cin >> x;
    cin >> y;

    if (x > y)
    {
        troca = x;
        x = y;
        y = troca;
    }
    soma = 0;
    for (i = x + 1; i < y; i++)
    {
        if (i % 2 != 0)
        {
            soma = soma + i;
        }
    }

    cout << "SOMA DOS IMPARES: " << soma << endl;

    return 0;
}

File: Java/Estrutura repetitiva/Para/Soma_impares.java
import java.util.Locale;
import java.util.Scanner;

public class Soma_impares {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int x, y, i, soma, troca;

        System.out.println("Digite dois numeros: ");
        x = sc.nextInt();
        y = sc.nextInt();

        if (x > y) {
            troca = y;
            y = x;
            x = troca;
        }

        soma = 0;

        for (i = x + 1; i < y; i++) {
            if (i % 2 != 0) {
                soma = soma + i;
            }
        }

        System.out.print("SOMA DOS IMPARES = " + soma);

        sc.close();

    }
}

